src/library.js

```diff
@@ -3617,6 +3617,8 @@ mergeInto(LibraryManager.library, {
   // have __heap_base hardcoded into it - it receives it from JS as an extern
   // global, basically).
   __heap_base: '{{{ HEAP_BASE }}}',
```

Comment:
do we still need `heap_base` here? (it is removed from `stack_limits.s` but I guess there's another use?)

Reply:
Yes, this is actually only used in the dynamic linking case.   If you looks this block is inside `#if RELOCATABLE`.  This is a symbol that wasm-ld normally provides but in the case of dynamic linking its expected that the runtime provides it.  

It is still used in emscripten, e.g. in sbrk.c